pkg/tsdb/prometheus/buffered/time_series_query.go

```diff
@@ -379,11 +379,7 @@ func matrixToDataFrames(matrix model.Matrix, query *PrometheusQuery, frames data
 
 		for i, k := range v.Values {
 			timeField.Set(i, k.Timestamp.Time().UTC())
-			value := float64(k.Value)
-
-			if !math.IsNaN(value) {
-				valueField.Set(i, value)
-			}
+			valueField.Set(i, float64(k.Value))
```

Comment:
@kylebrandt @itsmylife in order to bring buffered in line with the streaming response, i needed to stop skipping `NaN` in buffered responses. is there an issue here with alerting?

Reply:
@kylebrandt is the right person for this question.